src/jdk.hotspot.agent/doc/transported_core.html

```diff
@@ -10,101 +10,98 @@ <h1>Debugging transported core dumps</h1>
+to the debugger machine and included in <b>PATH</b>.
 </p>
 
+<h3>Using  transported core dumps on macOS</h3>
 <p>
-Support for transported core dumps is <b>not</b> built into the standard version of libproc.so. You need to
-set <b>LD_LIBRARY_PATH</b> env var to point to the path of a specially built version of libproc.so. 
-Note that this version of libproc.so has a special symbol to support transported core dump debugging. 
-In future, we may get this feature built into standard libproc.so -- if that happens, this step (of 
-setting LD_LIBRARY_PATH) can be skipped.
+SA normally uses the path to the specified java executable to locate the JDK libraries. It will look in the following subdirectories for them (relative to the path to the specified java executable): <b>../lib</b>, <b>../lib/server</b>, <b>../jre/lib</b>, and <b>../jre/lib/server</b>. If not found in any of those locations, it will look in the same subdirectories relative to the <b>JAVA_HOME</b> environment variable, but using <b>JAVA_HOME</b> normally should not be necessary.
 </p>
 
-<h3>Ignoring libthread_db.so failures</h3>
 <p>
-If you are okay with missing thread related information, you can set 
-<b>SA_IGNORE_THREADDB</b> environment variable to any value. With this
-set, SA ignores libthread_db failure, but you won't be able to get any
-thread related information. But, you would be able to use SA and get
-other information.
+For locating the user JNI libraries, SA uses <b>DYLD_LIBRARY_PATH</b>. It can contain more than one directory separated by a colon. <b>DYLD_LIBRARY_PATH</b> can also be used for locating the JDK libraries, but it needs to specify the full path to the libraries. SA will not automatically search subdirs such as <b>lib/server</b> as it does for <b>JAVA_HOME</b>.
 </p>
 
-<h3>Linux SA against transported core dumps</h3>
 <p>
-On Linux, SA parses core and shared library ELF files. SA <b>does not</b> use
-libthread_db.so or rtld_db.so for core dump debugging (although 
-libthread_db.so is used for live process debugging). But, you
-may still face problems with transported core dumps, because matching shared
-objects may not be in the path(s) specified in core dump file. To
-workaround this, you can define environment variable <b>SA_ALTROOT</b>
-to be the directory where shared libraries are kept. The semantics of
-this env. variable is same as that for Solaris (please refer above).
+For locating the macOS libraries, SA uses <b>SA_ALTROOT</b> similar to the linux support, except it does not use it to map all the subdirs. It just appends <b>SA_ALTROOT</b> to the full path of each macOS library. So if you specify <b>SA_ALTROOT=/altroot</b>, SA will prepend <b>/altroot</b> to the full path of each macOS library. Note however, due to <a href="https://bugs.openjdk.org/browse/JDK-8249779">JDK-8249779</a> , SA will not even try to open macOS libraries, so at the moment there is no need to try to match up the macOS libraries by pointing to them with <b>SA_ALTROOT</b>.
```

Comment:
Nit: too long line - hard to review.

Reply:
Ok, I split these all up into shorter lines. The rendered html is still the same.